src/app/app.component.ts

```diff
@@ -5,17 +5,45 @@
  */
 import { Component, OnInit } from '@angular/core';
 import { AnalyticsService } from './@core/utils/analytics.service';
+import { TranslateService } from '@ngx-translate/core';
 
 @Component({
   selector: 'ngx-app',
   template: '<router-outlet></router-outlet>',
 })
 export class AppComponent implements OnInit {
 
-  constructor(private analytics: AnalyticsService) {
+  constructor(
+    private translate: TranslateService,
+    private analytics: AnalyticsService) {
+      this.initTranslate();
   }
 
   ngOnInit(): void {
     this.analytics.trackPageViews();
   }
+
+
+  initTranslate() {
```

Comment:
I feel like this function could take a little cleanup in the comments :)

Reply:
I mean literaly remove the commented code. If its not used, then it shouldnt make its way to master